docset/windows/international/Copy-UserInternationalSettingsToSystem.md

```diff
@@ -0,0 +1,89 @@
+
+---
+ms.mktglfcycl: manage
+ms.sitesec: library
+ms.author: ertorres
+author: Erik Torres
```

Comment:
This should be a github username.

Reply:
@erik0686 The PR is good to go after applying the above suggestion. Thank you for your contributions.